core/storage/suggestion/gae_models.py

```diff
@@ -232,6 +241,34 @@ def query_suggestions(cls, query_fields_and_values):
 
         return query.fetch(feconf.DEFAULT_QUERY_LIMIT)
 
+    @classmethod
+    def get_translation_suggestions_with_exp_ids(cls, exp_ids):
+        """Gets all translation suggestions corresponding to explorations with
+        the given exploration ids.
+
+        Args:
+            exp_ids: list(str). List of exploration ids to query for.
+
+        Returns:
+            list(SuggestionModel). A list of translation suggestions that
+            correspond to the given exploration ids.
+        """
+
+        query = (
+            cls.get_all()
+            .order(cls.key)
+            .filter(cls.suggestion_type == SUGGESTION_TYPE_TRANSLATE_CONTENT)
+            .filter(cls.target_id.IN(exp_ids))
+        )
+        suggestion_models = []
+        cursor, more = (None, True)
+        while more:
+            results, cursor, more = query.fetch_page(
+                feconf.DEFAULT_QUERY_LIMIT, start_cursor=cursor)
+            suggestion_models.extend(results)
+        return suggestion_models
+
+
```

Comment:
Drop one newline -- methods in a class should be separated by 1 newline, not 2.

Reply:
Removed - thanks!